docs/proposals/28-autonomous-shoot-clusters.md

```diff
@@ -0,0 +1,497 @@
+However, the creation of such clusters by end-users is not covered by this GEP.
+This could become a future topic if we consider it relevant enough.
+
+Handling creation of autonomous shoot clusters by end-users requires refined access controls for the `gardenlet`.
+Currently, the `gardenlet` runs as part of a seed cluster and has access to some resources, which are not accessible to end-users but only to operators.
+An example resource is the `ControllerDeployment`, which may contain credentials, e.g., for certificate management or domain name services.
+In case this scenario is desired and the security boundary between Gardener operators and end-users should stay as is, the extension deployment needs to be adapted.
+
+While it may be appealing to have support for autonomous shoot cluster for end-users, it is important to understand that this feature may complicate the world for both operators and end-users.
+The benefits of autonomous shoot clusters for the garden runtime cluster and/or soil clusters, i.e., unmanaged seed clusters, is fairly obvious:
+It allows to create a Gardener landscape without external dependency to a Kubernetes service provider.
+However, the benefits for end-users are less clear (except for the mentioned edge/on-prem/air-gapped use cases).
+It needs to be carefully considered and documented for what scenarios autonomous shoot clusters are the right choice, and when ordinary shoot clusters with their control plane running on a seed cluster are the better way to go.
+Without this consideration, the feature may increase the operational complexity for Gardener operators, and the cognitive load for end-users.
+
+### Network Connectivity to Garden Cluster
+
+Another aspect that was only barely touched in this GEP is the reliability of the network connectivity between the autonomous shoot cluster and Gardener.
+In the section [`gardenadm connect`](#gardenadm-connect), it was mentioned that the cluster should function also when air-gapped/when it has limited or no network connectivity.
+However, we are deliberately vague what "limited network connectivity" means.
+Detailing this aspect out can be considered future work.
+As mentioned in the corresponding section, the [version skew](../deployment/version_skew_policy.md) plays an important role in this context, determining the maximum time the autonomous shoot cluster can be disconnected from Gardener.
+Long term support of Gardener and associated components may also be considered future work.
+
+### Changes to Shoot Components
+
+Due to the changed network layout of autonomous shoot clusters, the usage of some shoot components needs to be reconsidered.
+Examples are the VPN connection between control plane and data plane, the `apiserver-proxy` and the way metrics are scraped.
+This GEP does not cover these changes in detail, but they are important to be considered in the future.
```

Comment:
Just to be sure: is it planned to implement these changes as part of this GEP or only later on?
In my naive thinking, it doesn't make sense to eliminate VPN and apiserver-proxy later on because they will simply not work in autonomous clusters without the istio-ingressgateway counterpart on the seed.

Reply:
Removing non-required VPN components is part of this GEP, of course. I'll clarify this part.